integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/MessageOutController.java

```diff
@@ -175,6 +184,17 @@ public ResponseEntity uploadFiles(
         }
         ConversationResource conversationResource = find.get();
 
+        Optional<String> arkivmeldingFile = request.getFileMap().values().stream()
+                .map(MultipartFile::getOriginalFilename)
+                .filter(f -> ARKIVMELDING_FILE.equals(f))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1612)